server/models/db/workerTimeslot.js

```diff
@@ -21,7 +21,14 @@ async function updateWorkerAvailability(workerId, specificDate, newStatus) {
     return db.any(`update ${Tables.workerTimeslot} set status='${newStatus}' where date='${specificDate}' and worker_id='${workerId}';`);
 }
 
+//This method updates the worker avaiability table 
+async function updatingTheWorkerAvailability(status){
+    const queryCondition = `where w1.date='${specificDate}' and w1.worker_id='${workerId}'`;
```

Comment:
Hi Simran,

I do not believe that you need the `workerId` and `date` parameters for this query. I would recommend finding the status corrsponding to only the workerTimeslotId parameter in the Worker_Timeslot table. You can look at the     `server/migrations/009.do.create-worker-timeslot-table.sql` for reference.

Reply:
Hi Nancy! 

Thank you very much for your comments! Peter and I have amended the changes to reflect your comments in the latest commits. I appreciate all your help and time! 

Simran

Reply:
Awesome! I'm glad you were able to address this!